packages/app/src/runner/selector-playground/Highlight.ce.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    ref="highlightRef"
+    class="highlight"
+    :style="style"
+  />
+  <div
+    ref="tooltipRef"
+    class="tooltip"
+  >
+    <span>{{ selector }}</span>
+    <div
+      ref="arrow"
+      class="arrow"
+    />
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { computePosition, flip, offset, arrow } from '@floating-ui/dom'
+import { StyleValue, onMounted, nextTick, ref, Ref } from 'vue'
+
+defineProps <{
+  selector: string
+  style: StyleValue
+}>()
+
+const highlightRef: Ref<HTMLElement | null> = ref(null)
+const tooltipRef: Ref<HTMLElement | null> = ref(null)
+
+onMounted(() => {
```

Comment:
Both the React and Vue version of this  manipulate the DOM so heavily... we need `ref` and `nextTick` a lot, seems like a code smell.

Do you think we should just do this without a UI lib and just use regular JS/TS?

Reply:
WDYT @sainthkh

Reply:
I tried it and did it. It became much simpler!

Reply:
I'm sorry. I wasn't sure if it was possible. So, I tested it first and learned that it is doable. As our tooltip is simple, we don't need a heavy tool like `floating-ui`. 

I'm not sure we also don't need it for `studio` UIs. But it's not necessary for `Highlight`.

Reply:
Neat - I had a feeling the UI lib (react, vue) were both making this more complex than necessary.